stock_inventory_virtual_location/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': "Stock Inventory Virtual Location",
+    'summary': "Allows to change the virtual location"
+               " in inventory adjustments.",
+    'author': 'Eficent, Odoo Community Association (OCA)',
+    'website': "https://github.com/OCA/stock-logistics-warehouse",
+    'category': 'Warehouse Management',
```

Comment:
Could you set this to 'Warehouse' as it won't appear in OCA shop in the good category?

Reply:
done for all modules